.cirrus.yml

```diff
@@ -327,6 +327,7 @@ alt_build_task:
 osx_alt_build_task:
     name: "OSX Cross"
     alias: osx_alt_build
+    only_if: $CI != $CI  # Temporarily disabled while infra. non-functional
```

Comment:
Anything that says "new" or "temporary" or "experimental", I like to add an explicit YYYY-MM-DD